demisto_sdk/commands/common/constants.py

```diff
@@ -841,11 +845,15 @@ class PB_Status:
     'ip': {'IP.Address'},
     'url': {'URL.Data'}
 }
-
-PACK_INITIAL_VERSION = '1.0.0'
-PACK_SUPPORT_OPTIONS = ['xsoar', 'partner', 'developer', 'community']
 XSOAR_SUPPORT = "xsoar"
 XSOAR_AUTHOR = "Cortex XSOAR"
+PACK_INITIAL_VERSION = '1.0.0'
+PACK_SUPPORT_OPTIONS = ['xsoar', 'partner', 'developer', 'community']
+LEVEL_SUPPORT_OPTIONS_1 = [XSOAR_SUPPORT]
+LEVEL_SUPPORT_OPTIONS_2 = [XSOAR_SUPPORT, CERTIFIED_PARTNER]
+LEVEL_SUPPORT_OPTIONS_3 = [XSOAR_SUPPORT, CERTIFIED_PARTNER, PARTNER]
+LEVEL_SUPPORT_OPTIONS_4 = [XSOAR_SUPPORT, CERTIFIED_PARTNER, PARTNER, COMMUNITY]
```

Comment:
There might be an easier way to do this, using the preset error_ignore flags.
```
PRESET_ERROR_TO_IGNORE = {
    'community': ['BC', 'CJ', 'DS', 'PA117'],
    'non-certified-partner': ['CJ']
}
```
see in errors.py